src/js/common/polyfills.js

```diff
@@ -8,17 +8,9 @@
 import 'babel-polyfill';
 
 // Basic polyfills.
-// TODO(awong): These do NOT correctly conditionally load the polyfill.
-// The polyfill is always loaded. require.ensure() should be used instead but
-// then load ordering needs to be worked out. Fix later.
-import Modernizr from 'modernizr';
```

Comment:
Modernizr is not necessary, as far as I can tell. I need to do a little more research to be sure.